File: src/Conferences/HexMaster.Attendr.Conferences/ListConferences/ConferenceListItemDto.cs
namespace HexMaster.Attendr.Conferences.ListConferences;

public sealed record ConferenceListItemDto(
    Guid Id,
    string Title,
    string? City,
    string? Country,
    DateOnly StartDate,
    DateOnly EndDate,
    string? ImageUrl,
    bool HasSynchronizationSource,
    int SpeakersCount,
    int RoomsCount,
    int PresentationsCount
);

File: src/App/src/app/shared/models/conference-list-item-dto.ts
export interface ConferenceListItemDto {
    id: string;
    title: string;
    city: string;
    country: string;
    startDate: string;
    endDate: string;
    imageUrl?: string;
    isVisible: boolean;
    speakersCount: number;
    roomsCount: number;
    presentationsCount: number;
}
